extensions/adobe/experience/offer-management/content/html-content.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/html-content",
+  "meta:extends": ["https://ns.adobe.com/experience/offer-management/representation-content"],
+  "meta:abstract": false,
+  "meta:extensible": true,
+  "type": "object",
+  "title": "HTML Content",
+  "description": "HTML content used in an offer representation",
+  "definitions": {
+    "html-content": {
```

Comment:
a "blob of HTML" is not a valid thing to be storing in an XDM.  This should be a URL to the actual storage.  (that also addresses linked/reference elements like CSS, images, etc.)

Reply:
This HTML is a fragment or content component not a page or a complete asset. I think there is more definition required for the schemas that are currently located under xdm/schemas/content
Those things are not easy to take out of their page container and not all fragments are stored as html page like documents. But if you look closely the representation-content.schema.json here is trying to align with the `https://ns.adobe.com/xdm/content/page-component#/definitions/component` there.
Most notably is the usage of the `@type` property as the processing hint and a way to identify the component type. In the schema here (offer representation-content) the values of the `@type` property are an enumerated restriction of the `@type`'s 
```
      "properties": {
        "@type": {
          "type": "string",
          "format": "uri",
          "description":
            "Type of the component. Acts as processing hint for the client."
        }
      }
```
I'm not saying we shouldn't allow the inclusion of content repository managed content. It would be added at a later point. We have that on our roadmap.